tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/pcluster.config.yaml

```diff
@@ -0,0 +1,42 @@
+Region: {{ region }}
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: c5n.18xlarge
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore # Required to report patching status
+      - Policy: arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess #Required to read software artifacts from s3
+Scheduling:
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: q1
+      ComputeResources:
+        - Name: c5n-18xl-efa
+          InstanceType: c5n.18xlarge
+          MinCount: {{ number_of_nodes }} # 8
```

Comment:
MinCount and Maxcount are both set to number_of_nodes, that is always 32, so 8 is wrong in this comment.